modules/programs/zsh.nix

```
@@ -60,6 +60,15 @@ let
         description = "History file location";
       };
 
+      ignorePattern = mkOption {
+        type = types.nullOr types.string;
+        default = null;
+        description = ''
+          Do not enter command lines into the history list
+          if they matches the given shell pattern.
```

Small typo:
```suggestion
          if they matche the given shell pattern.
```

---

@berbiche isn't your suggestion also a typo?

```suggestion
          if they match the given shell pattern.
```